qiskit_metal/renderers/renderer_ansys/ansys_renderer.py

```diff
@@ -122,6 +123,7 @@ class QAnsysRenderer(QRendererAnalysis):
         Cj=0,  # Cj *must* be 0 for pyEPR analysis! Cj has units of femtofarads (fF)
         _Rj=0,  # _Rj *must* be 0 for pyEPR analysis! _Rj has units of Ohms
         max_mesh_length_jj='7um',  # maximum mesh length for Josephson junction elements
+        max_mesh_length_port='7um', # maximum mesh length for Ports in Eigenmode Simulations
```

Comment:
I shall remove the variable from Q3D, and add a commit.

Reply:
@ThomasGM4 I tried to remove the options from ansys_renderer, but it seems that that is causing a plethora of errors. In my opinion, it would be best to retain it.

Reply:
Huh, I wonder what could be causing those errors. For the moment can just leave it be, and have a vestigial option with Q3D simulations.